paper/main.tex

```diff
@@ -924,107 +938,99 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
+too provocable is when the matches are not long. Forgiveness as defined by R.
+Axerlod was not explored here. This was mainly because the two round states were
+not recorded during the data collection. That is something we would aim to
+incorporate in future work.
+The features analysis also concluded that
+there is a significant importance in \textbf{adapting to the environment}, and
+more specifically, to the mean cooperator. Moreover, the manner in which a
+strategy achieves a given cooperation rate relative to the tournament population
+average is important. In standard tournaments a strategy would aim to be the
+mean the cooperator while in noisy tournaments the best performing players
+cooperate a lower rate than the tournament population on average.
+
+This could potentially explain the early success of TFT. TFT naturally achieves
+a cooperation rate near $C_{\text{mean}}$ by virtue of copying its opponent's
+last move while also minimizing instances where it is exploited by an opponent
+(cooperating while the opponent defects), at least in non-noisy tournaments. It
+could also explain why Tit For \(N\) Tats does not fare well for $N > 1$ -- it
+fails to achieve the proper cooperation ratio by tolerating too many defections.
+
+Similarly, our results could suggest an explanation regarding the intuitively
+unexpected effectiveness of memory-one strategies historically. Given that among
+the important features associated with success are the relative cooperation rate
+to the population average and the four memory-one probabilities of cooperating
+conditional on the previous round of play, these features can be optimized by a
+memory-one strategy such as TFT. Usage of more history becomes valuable when
+there are exploitable opponent patterns. This is indicated by the importance of
+SSE as a feature, showing that the first-approximation provided by a memory-one
+strategy is no longer sufficient.
+
+These results highlight a central idea in evolutionary game theory in this
+context: the fitness landscape is a function of the population (where fitness in
+this case is tournament performance). While that may seem obvious now, it shows
+why historical tournament results on small or arbitrary populations of
+strategies have so often failed to produce generalizable results.
+
+Overall, the five properties successful strategies need to have in a IPD competition
+based on the analysis that has been presented in this manuscript are:
 
 \begin{itemize}
-\item be nice, if the environment isn't noisy and the games are long
-    \item be somewhat provocable,
-    \item be a little envious,
-    \item be clever,
-    \item adapt to the environment (including the population of strategies).
+    \item Be ``nice'' in non-noisy environments or when game lengths are longer
+    \item Be provocable but also generous when game lengths are longer
+    \item Be a little bit envious
+    \item It's ok to be clever
```

Comment:
```suggestion
    \item Be clever
```

(This is to match the grammar of the preceding statement and the other bullet points.)